package.json

```diff
@@ -130,5 +130,8 @@
       "text",
       "cobertura"
     ]
+  },
+  "resolutions": {
+    "@types/react": "^18.2.12"
   }
```

Comment:
This would fix the lint issues that were present in the build before this change. Is that acceptable?

Reply:
it was enought to update the yarn lock file